src/pages/Kakaologin/Kakaologin.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export default function Kakaologin() {
+  return (
+    <div>
+      <Component>
+        <span>test</span>
+      </Component>
+    </div>
+  );
+}
+
+const Component = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  height: 700px;
+  background-color: white;
+`;
```

Comment:
```suggestion
export default function KakaoLogin() {
  return (
      <Container>
        <span>test</span>
      </Container>
  );
}

const Container = styled.div`
  height: 700px;
  background-color: white;
`;
```
- 카멜 케이스 적용해주세요
- 저는 가장 바깥쪽 요소는 고민 없이 Container로 정하는 편입니다
- box-sizing은 common에서 설정하고 더 이상 건드릴 일이 없습니다
- div는 가로 100%입니다

Reply:
브랜치 새로 생성 도중 테스트 파일을 만들어 코드 복사를 했던부분을 간과한것같습니다! 리뷰 사항 참고하여 새롭게 코드 작성하도록 하겠습니다!! 감사합니당